scripts/prepare-for-release

```diff
@@ -20,7 +20,7 @@
 # NOTE: only the files updated by this script is included in the commit / PR i.e. release version updates in $FILES
 # This script is NOT intended to be combined with other unmerged local changes
 
-set -o pipefail
+set -euo pipefail
```

Comment:
This looks really good! 

Could the hardcoded AEMM paths (I think that's just the helm stuff) be generalized as parameters to a helm chart dir?  That way it would be easy to port this to other projects.

Reply:
discussed offline. will generalize the script at a  later time.